rust/Earthfile

```diff
@@ -6,27 +6,33 @@ VERSION --global-cache 0.7
 #   - sweep_days (4): +CARGO calls use cargo-sweep to clean build artifacts that haven't been accessed for this number of days.
 INIT:
     COMMAND
-    RUN if [ -f /earthly/cfg/cache_id ]; then \
+    RUN if [ -f /tmp/earthly/cfg/cache_id ]; then \
```

Comment:
Should `/earthly_lib_rust_temp` move too?
Also I still see a `ENV CARGO_HOME="/earthly/.cargo"`

Reply:
Fixed